src/lib/ckeditor/config.js

```diff
@@ -22,7 +22,8 @@ CKEDITOR.editorConfig = function( config ) {
     config.autoGrow_onStartup = true;
     config.startupFocus = 'end';
     config.mathJaxLib = 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
-    config.allowedContent = true;
+	config.allowedContent = true;
+	config.extraAllowedContent = 'mms-run-xlr';
```

Comment:
not needed, since allowedContent is already true

Reply:
hmm i'll look at it again... i think the editor was clearing the custom tag in source